eg3.py

```diff
@@ -0,0 +1,31 @@
+"""You are given a list of sentences. Create a word tree from each sentence and find how many times the word is appeared in other word trees."""
+
+#list created with sentences
+sentences = ["My name is Ram", "He is a good person", "You should be careful while coding",
+             "He can do better", "The person is mysterious", "Jay Shree Ram", "It is my pen"]
+
+
+SplitSentences = []
```

Comment:
For each PR, please make sure you declare the variable names in smaller cases, as it is the correct way to declare the names. See the python standard rules for naming conventions.